django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -68,7 +73,10 @@ def disallowed_aggregates(self):
             models.Union,
         ]
         is_mariadb = self.connection.mysql_is_mariadb
-        if is_mariadb or self.connection.mysql_version < (8, 0, 24):
+        if is_mariadb:
+            if self.connection.mysql_version < (11, 7):
```

Comment:
```suggestion
        if is_mariadb and self.connection.mysql_version < (11, 7):
```

Reply:
This won't work as "elif" should be check only for MySQL.